[Docs] Minor doc fixes for init_process_group

osalpekar opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#47644 [Docs] Minor doc fixes for init_process_group**

Minor Update to the init_process_group docs.

Differential Revision: [D24633432](https://our.internmc.facebook.com/intern/diff/D24633432/)

osalpekar committed:
[Docs] Minor doc fixes for init_process_group

Minor Update to the init_process_group docs.

Differential Revision: [D24633432](https://our.internmc.facebook.com/intern/diff/D24633432/)

[ghstack-poisoned]

mrshenli commented on torch/distributed/distributed_c10d.py:
```diff
@@ -401,11 +401,11 @@ def init_process_group(backend,
             will provide errors to the user which can be caught and handled,
             but due to its blocking nature, it has a performance overhead. On
-            the other hand, ``NCCL_ASYNC_ERROR_HANDLING`` has little
+            the other hand, ``NCCL_ASYNC_ERROR_HANDLING`` has  very little
```
redundant space before "very"

osalpekar committed:
Update on "[Docs] Minor doc fixes for init_process_group"


Minor Update to the init_process_group docs.

Differential Revision: [D24633432](https://our.internmc.facebook.com/intern/diff/D24633432/)

[ghstack-poisoned]

osalpekar committed:
Update on "[Docs] Minor doc fixes for init_process_group"


Minor Update to the init_process_group docs.

Differential Revision: [D24633432](https://our.internmc.facebook.com/intern/diff/D24633432/)

[ghstack-poisoned]
